### PR TITLE
etc/kamailio.cfg: detect sipvicious as scanner

### PR DESCRIPTION
**Pull request by kevin-olbrich**
#### Pre-Submission Checklist

- [X] Commit message has the format required by CONTRIBUTING guide
- [X] Commits are split per component (core, individual modules, libs, utils, ...)
- [X] Each component has a single commit (if not, squash them into one commit)
- [X] No commits to README files for modules (changes must be done to docbook files
in `doc/` subfolder, the README file is autogenerated)

#### Type Of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:

- [ ] PR should be backported to stable branches
- [X] Tested changes locally
- [ ] Related to issue #XXXX (replace XXXX with an open issue number)

### CODE REVIEW

**Comment by sergey-safarov** [comment]
Probable you want look other [scanners UA](https://github.com/2600hz/kazoo-configs-kamailio/blob/master/kamailio/default.cfg#L402-L407).

**Comment by miconda** [comment]
Thanks!

**Comment by kevin-olbrich** [comment]
> Probable you want look other [scanners UA](https://github.com/2600hz/kazoo-configs-kamailio/blob/master/kamailio/default.cfg#L402-L407).

IMHO adding these makes sense. What do you think?

**Comment by henningw** [comment]
To add the missing user agents from the kazoo cfg snippet sounds good to me.

**Comment by miconda** [comment]
I don't think it is good to add a lot of string there. The default configuration files is more like a reference. sipvicious is a well known one and made sense (although most of its versions use `friendly-scanner` in user-agent).

But, afaik, portsip is an sdk and pbx. Then, others like NiceGuy likely to be sipvicious with a different value set for user agent. If the attacker was changing the default value to something else, banning that one by regex won't get it fixed for too long, the attacker will use a different value soon.

In other words, if it is something very common or a tool used for scanning attacks, makes sense to add. If it is just different user agent name used by an attacker against a platform, it should not be added in the default config file. There is pike+htable that can detect DoS attacks.
